### PR TITLE
EOS-17357 : Initiate multipart STs and UTs

### PR DESCRIPTION
**Pull request by pranavPawar07**
Signed-off-by: Pranav Diliprao Pawar <pranav.pawar@seagate.com>

### CODE REVIEW

File: st/clitests/jclient_spec.py [hunk 1]
```diff
@@ -756,7 +756,20 @@ def get_response_elements(response):
 
     JClientTest('Jclient can delete bucket').delete_bucket("seagatebucket").execute_test().command_is_successful()
 
+# ************ TEST: Multiple INIT-MPU ***********
```
Comment (knrajnambiar76) [inline st/clitests/jclient_spec.py:759]: TEST: Multipart initiate of same object twice

File: st/clitests/jclient_spec.py [hunk 2]
```diff
@@ -756,7 +756,20 @@ def get_response_elements(response):
...
+    JClientTest('Jclient can create bucket seagatebucket').create_bucket("seagatebucket").execute_test().command_is_successful()
+
+    JClientTest('Jclient can initiate multipart upload').init_mpu("seagatebucket", "18MBfile", 18000000)\
+            .execute_test().command_is_successful()
+
+    JClientTest('Jclient can again initiate multipart upload').init_mpu("seagatebucket", "18MBfile", 18000000)\
+            .execute_test().command_is_successful()
 
+    result = JClientTest('Jclient can list all multipart uploads.').list_multipart("seagatebucket").execute_test()
+    result.command_response_should_have('18MBfile')
```
Comment (knrajnambiar76) [inline st/clitests/jclient_spec.py:769]: command_response_should_have('18MBfile')  A new function you can define, ie command_response_should_have_n_times('18MBfile',2) that function sees whether 18MBfile occured 2 times

File: st/clitests/framework.py [hunk 3]
```diff
@@ -161,6 +161,22 @@ def command_response_should_have(self, msg):
         print("Response has [%s]." % (msg))
         return self
 
+    def command_response_should_have_n_times(self, msg, count):
```
Comment (knrajnambiar76) [inline st/clitests/framework.py:164]: change msg to filename

File: st/clitests/framework.py [hunk 4]
```diff
@@ -161,6 +161,22 @@ def command_response_should_have(self, msg):
...
+        temp_count = 0
+        if not Config.dummy_run:
+            #counting the number of occurences of msg("should be equal to 'count' times")
+            text = self.status.stdout
+            index = 0
+            while index < len(text):
+                index = text.find(msg, index)
+                if index == -1:
+                    break
+                index += 8
```
Comment (knrajnambiar76) [inline st/clitests/framework.py:174]: dont hard code it, you need to get length of the filename passed

File: st/clitests/jclient_spec.py (hunk 1 above)
Comment (pranavPawar07) [reply st/clitests/jclient_spec.py]: done

File: st/clitests/jclient_spec.py (hunk 2 above)
Comment (pranavPawar07) [reply st/clitests/jclient_spec.py]: done

File: st/clitests/framework.py (hunk 3 above)
Comment (pranavPawar07) [reply st/clitests/framework.py]: done

File: st/clitests/framework.py (hunk 4 above)
Comment (pranavPawar07) [reply st/clitests/framework.py]: done

**Review by knrajnambiar76 — APPROVED** [review]

**Review by DPG17 — APPROVED** [review]

**Pull request merged**